fragments/maintenance/backend.php

```diff
@@ -3,7 +3,15 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Maintenance</title>
+    <title>
+        <?php
+            if (rex_addon::get('yrewrite')->isAvailable() && rex_yrewrite::getCurrentDomain()->getName() !== '') {
```

Comment:
```suggestion
            if (rex_addon::get('yrewrite')->isAvailable() && rex_yrewrite::getCurrentDomain()->getName() !== null) {
```

@alxndr-w Wir hatten es Ja neulich im Slack darüber ob `getName()` auch Null zurückgeben kann. Dann würde es eventuell sinn machen, anstatt auf leer, auf `null` zu prüfen.

Wie siehst du das?

Reply:
```suggestion
            if (rex_addon::get('yrewrite')->isAvailable() && rex_yrewrite::getCurrentDomain() !== null && rex_yrewrite::getCurrentDomain()->getName() !== '') {
```

Reply:
Oder so.. doppelt hält besser ;)

Reply:
So meinte ich. Du musst sicherstellen, dass bei `rex_yrewrite::getCurrentDomain()` auch wirklich ein Objekt zurückkommt, bevor du die Methode `->getName()` ausführen kannst.

Bzw. es gibt auch noch eine andere Schreibweise:
`rex_yrewrite::getCurrentDomain()?->getName()`
`                                ^           `

ich weiß nicht, ob die bereits in PHP 8.1 verfügbar ist. Hier könntest du mal recherchieren:

https://stitcher.io/blog/php-8-nullsafe-operator

Reply:
Kam wohl mit PHP 8.0 hinzu: https://php.watch/versions/8.0/null-safe-operator